python/lsst/sims/catalogs/utils/testUtils.py

```diff
@@ -169,8 +168,7 @@ def makeGalTestDB(filename='testDatabase.db', size=1000, seedVal=None,
     for i in range(size):
         period = rng.random_sample(1)[0]*490. + 10.
         amp = rng.random_sample(1)[0]*5. + 0.2
-        varParam = {'varMethodName': 'testVar', 'pars': {'period': period, 'amplitude': amp}}
-        paramStr = json.dumps(varParam)
+        paramStr = '{"varMethodName": "testVar", "pars": {"period": %.6e, "amplitude": %.6e}}' % (period, amp)
```

Comment:
Maybe add a comment here that you are formatting the random numbers to a fixed string length?